.github/workflows/gradle.yml

```diff
@@ -1159,7 +1158,7 @@ jobs:
   helm-acceptance-test:
     name: "Platform: Acceptance Tests (Helm)"
     # In case of self-hosted EC2 errors, removed the `needs` line and switch back to running on ubuntu-latest.
-    needs: [start-helm-acceptance-test-runner] # required to start the main job when the runner is ready
+    needs: [ start-helm-acceptance-test-runner ] # required to start the main job when the runner is ready
```

Comment:
Is this change expected? Shouldn't it be
```
needs:
     - start-helm-acceptance-test-runner
```

Reply:
This was the formatter adding some spaces between the square-braces.  From the yaml parsing POV, there isn't a difference between
```yml
needs: [ value ]
```
and 
```yml
needs:
  - value
```